package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tram-one",
-  "version": "10.1.11",
+  "version": "11.0.0",
```

Comment:
Major bump. In reality, this might not be breaking for JS projects, but anyone using Typescript would start getting compile time errors / warnings, so for that reason, it should be breaking.